tests/integration/template_weekly_comprehensive_v3.cfg

```diff
@@ -87,20 +87,15 @@ walltime = "00:30:00"
 
 [e3sm_to_cmip]
 active = True
+frequency = "monthly"
 ts_num_years=2
 walltime = "00:30:00"
 
   [[ atm_monthly_180x360_aave ]]
-  frequency = "monthly"
   input_files = "eam.h0"
```

Comment:
The updates look good. Thank you. I'm curious if `input_files` is required in this sub-session?

Reply:
It's used in `e3sm_to_cmip.bash`:
```
  {% if input_files.split(".")[0] == 'clm2' or input_files.split(".")[0] == 'elm' -%}
  --var-list \
  '{{ cmip_var_lnd }}' \
  --realm \
  lnd \
  {% endif -%}
  {% if input_files.split(".")[0] == 'cam' or input_files.split(".")[0] == 'eam' -%}
  --var-list \
  '{{ cmip_var_atm }}' \
```

Reply:
Oh, I see, it is used for determine `realm` and `var-list` accordingly. Thanks.